src/App.tsx

```diff
@@ -1,19 +1,175 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React, { useContext, useEffect, useRef } from 'react';
+import React, {
+  useContext, useEffect, useRef, useState,
+} from 'react';
+import classNames from 'classnames';
 import { AuthContext } from './components/Auth/AuthContext';
+import {
+  getTodos, addTodo, toggleStatus, removeTodo,
+} from './api/todos';
+import { Todo } from './types/Todo';
+
+export enum FilterStatus {
+  All = 'all',
+  Active = 'active',
+  Completed = 'completed',
+}
+
+type Title = {
+  title?: null | string
+};
 
 export const App: React.FC = () => {
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [todoTitle, setTodoTitle] = useState('');
+  const [isHidden, setIsHidden] = useState(true);
+  const [filterStatus, setFilterStatus] = useState('all');
```

Comment:
its better to use enum here 
```suggestion
  const [filterStatus, setFilterStatus] = useState(FilterStatus.All);
```